internal/querynode/segment_loader.go

```diff
@@ -281,14 +281,18 @@ func (loader *segmentLoader) loadSegmentInternal(segment *Segment,
 	} else {
 		log.Debug("loading bloom filter...", zap.Int64("segmentID", segmentID))
 		pkStatsBinlogs := loader.filterPKStatsBinlogs(loadInfo.Statslogs, pkFieldID)
-		err = loader.loadSegmentBloomFilter(segment, pkStatsBinlogs)
-		if err != nil {
+		if err = loader.loadSegmentBloomFilter(segment, pkStatsBinlogs); err != nil {
 			return err
 		}
 	}
 
-	log.Debug("loading delta...", zap.Int64("segmentID", segmentID))
-	err = loader.loadDeltaLogs(segment, loadInfo.Deltalogs)
+	// Load delta logs for sealed segments.
```

Comment:
we need to add unit test on this path

Reply:
Since I'm not sure if growing segment load delta log is fully tested. Such a bug!